%3a133/vectors.sls

```diff
@@ -17,7 +17,7 @@
 
     ;;; * Mutators
     vector-swap! vector-fill! vector-reverse! vector-copy! vector-reverse-copy!
-    vector-reverse! vector-unfold! vector-unfold-right!
+    vector-unfold! vector-unfold-right!
```

Comment:
`vector-reverse!` is exported twice. Not sure if spec allows, but my version of psyntax does not like it.